save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -75,10 +75,12 @@ class VulnerabilityController(
         description = "Save vulnerability.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully saved vulnerability")
+    @PreAuthorize("permitAll()")
```

Comment:
is it a temporary solution? or everyone can save it?

Reply:
Yes. Anyone logging in can create a new vulnerability.